### PR TITLE
Error explanation for unauthorized domain access

### LINKED ISSUE #1023: `now alias` throws error on .es domains: WHOIS error
Hi,

the problem seems to be related to _es_ domains.
I have set on my domain the nameservers to Zeit's however a `now alias` fails gracefully:

```

now alias -d https://out-atuemambeo.now.sh vroodl.es
> [debug] GET /v2/now/deployments: 1037.283ms
> [debug] matched deployment iiecnh8yeLymAds4y5DFBuEX by url out-atuemambeo.now.sh
> vroodl.es is a custom domain.
⠇ Fetching domain info> [debug] #1 GET /domains/status?name=vroodl.es: 637.552ms
> Verifying the DNS settings for vroodl.es (see https://zeit.world for help)
> [debug] #1 GET /domains/price?name=vroodl.es: 767.304ms
> [debug] bailing on get domain due to 404
> [debug] #1 GET /whois-ns: 201.909ms
> [debug] Retrying: Error: Invalid whois response
Error: Invalid whois response
    at module.exports.EventEmitter.getNameservers.Promise.retry (/snapshot/now-cli/dist/now.js:811:19)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
> [debug] #2 GET /whois-ns-fallback: 2990.622ms
> [debug] Retrying: Error: Whois error (500): An unexpected internal error occurred
Error: Whois error (500): An unexpected internal error occurred
    at module.exports.EventEmitter.getNameservers.Promise.retry (/snapshot/now-cli/dist/now.js:821:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
> [debug] #3 GET /whois-ns-fallback: 2930.091ms
> [debug] Retrying: Error: Whois error (500): An unexpected internal error occurred
Error: Whois error (500): An unexpected internal error occurred
    at module.exports.EventEmitter.getNameservers.Promise.retry (/snapshot/now-cli/dist/now.js:821:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
> [debug] #4 GET /whois-ns-fallback: 2426.949ms
> Error! Unknown error: Error: Whois error (500): An unexpected internal error occurred
Error: Whois error (500): An unexpected internal error occurred
    at module.exports.EventEmitter.getNameservers.Promise.retry (/snapshot/now-cli/dist/now.js:821:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

I have already asked help at the zeit slack chat, however it seems to be a bug.

Best

### PR DESCRIPTION
**Pull request by timneutkens**
Fixes #1023

### CODE REVIEW

File: errors/unauthorized-domain.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Not authorized to access domain
+
+#### Why This Error Occurred
+
+You tried to add or update domain data, but you don't have access to the domain.
```
Comment (TooTallNate) [inline errors/unauthorized-domain.md]: Maybe:

> You tried to add or update a domain's configuration, but you don't have permission to modify the domain.

File: errors/unauthorized-domain.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+#### Possible Ways to Fix It
+
+If you/your team owns the domain you are most likely in the wrong context. Use `now switch` to select the team or user that owns the domain.
```
Comment (TooTallNate) [inline errors/unauthorized-domain.md]: Maybe:

> If you or your team owns the domain, then you are most likely in the wrong context.

**Comment by leo** [comment]
We need to ensure it gets logged using [this](https://github.com/zeit/now-cli/blob/canary/src/util/output/error.js)! 🙏

**Comment by timneutkens** [comment]
@leo nice, i didn’t know that was a thing, Will fix it later today 👌🏻

**Pull request merged**